lib/DB/Schema/Result/Course.pm

```diff
@@ -66,16 +66,16 @@ __PACKAGE__->set_primary_key('course_id');
 
 # set up the many-to-many relationship to users
 __PACKAGE__->has_many(course_users => 'DB::Schema::Result::CourseUser', 'course_id');
-__PACKAGE__->many_to_many(users => 'course_users', 'users');
+__PACKAGE__->many_to_many(users => 'course_users', 'user');
```

Comment:
This should remain plural.

Reply:
I was using the example in https://metacpan.org/pod/DBIx::Class::Relationship#many_to_many

It seems that the 3rd argument is the same as the belongs_to in the bridging table. I don't think this is an accessor name. The example cited above used singular for this relationship (actor and role).